src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX = 101;
```

Comment:
I'd check the requirements again :)
```suggestion
    private static final int MAX = 100;
```